src/views/Tree.vue

```diff
@@ -0,0 +1,159 @@
+<template>
+  <div class="c-tree">
+    <!-- TODO: add progress bar which displays with {{ isLoading }} -->
+    <ul v-for="workflow in workflows" :key="workflow.id">
+      <li v-if="workflow.name === workflowId">
+        <!-- Workflow -->
+        <span>
+          {{ workflow.name }}
+        </span>
+        <ul>
+          <li v-for="cycle in cycles" :key="cycle">
+            <!-- Cycle -->
+            <span>
+              {{ cycle }}
+            </span>
+            <ul v-for="proxy in workflow.taskProxies" :key="proxy.id">
+              <li v-if="proxy.cyclePoint === cycle">
```

Comment:
Will need to consider the efficiency of this approach, not sure how to test. If Vue does it's job right I think there may be a slight performance hit on initial-render but each following addition should be quick.